lib/DDG/Spice/What3Words.pm

```diff
@@ -14,7 +14,7 @@ spice alt_to => {
     what3words_arcgis => {
         to => 'http://geocode.arcgis.com/arcgis/rest/services/World/GeocodeServer/reverseGeocode?location=$1&distance=5000&f=json'
     },
-    standard_blend => {
+    what3words_standard_blend => {
         to => 'https://api.what3words.com/v2/standardblend?addr=$1&lang=en&key={{ENV{DDG_SPICE_W3W_APIKEY}}}&callback={{callback}}'
```

Comment:
@vovkman you'll want to remove `&callback={{callback}}` -- that's forcing the API call to return a JS functino call, instead of JSON